src/Main.java

```diff
@@ -0,0 +1,62 @@
+import task.BusinessTrip;
+
+
+//this is the code from stage 2. It's the same as on stage 1.
+//Since all the requirements from stage 2-file were already met - i didn't change anything.
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        BusinessTrip[] BusinessTrips = new BusinessTrip[]{
```

Comment:
```suggestion
        BusinessTrip[] dusinessTrips = new BusinessTrip[]{
```
нарушение code conventions.